dart/lib/src/isolate_error_integration.dart

```diff
@@ -37,6 +37,7 @@ RawReceivePort _createPort(Hub hub, SentryOptions options) {
 }
 
 /// Parse and raise an event out of the Isolate error.
+@internal
 @visibleForTesting
```

Comment:
is it common to use both of them together?

Reply:
Let me double check this